packages/ra-ui-materialui/src/form/SimpleForm.tsx

```diff
@@ -62,7 +62,12 @@ export const SimpleForm = (inProps: SimpleFormProps) => {
     return (
         <Form {...rest}>
             <Component className={className} sx={sx}>
-                <Stack alignItems="flex-start" {...sanitizeRestProps(props)}>
+                <Stack
+                    {...sanitizeRestProps(props)}
+                    sx={{
+                        alignItems: 'flex-start',
+                    }}
+                >
```

Comment:
I'm not sure about this. How can I override the `sx` now?

Reply:
Hi @djhi,

In the old sanitizeRestProps function, the sx property is already sanitized out https://github.com/marmelab/react-admin/blob/next/packages/ra-ui-materialui/src/form/SimpleForm.tsx#L99 

That means, in the original code it does not support overriding sx.

Reply:
We did not introduce or remove any new feature

Reply:
My mistake. Thanks